saleor/graphql/account/bulk_mutations/customer_bulk_update.py

```diff
@@ -96,6 +97,16 @@ class Meta:
         doc_category = DOC_CATEGORY_USERS
         permissions = (AccountPermissions.MANAGE_USERS,)
         error_type_class = CustomerBulkUpdateError
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.CUSTOMER_UPDATED,
+                description="A new customer account was updated.",
```

Comment:
Small change, but it's not "new" if we're updating it right?
```suggestion
                description="A customer account was updated.",
```